spec/factories/payloads/pull_request.rb

```diff
@@ -40,6 +40,34 @@
     end
     initialize_with { attributes.deep_stringify_keys }
 
+    trait :edited do
+      transient do
+        changed { 'title' }
+        from { 'Old PR Title' }
+      end
+
+      action { 'edited' }
+      changes do
+        {
+          changed => {
```

Comment:
```suggestion
          changed: {
```

Reply:
It doesn't work the same way. I want the key to be whatever is in the `changed` variable (`'title'` is the default). If I use `changed:`, the key will be the symbol `:changed`.

Reply:
you are right, my bad 😄